### ISSUE TITLE
text2art is not defined

### ISSUE THREAD

**Issue by KrAsH-CoD3** [sepandhaghighi/art#222]
#### Description
I can not make use of text2art

#### Steps/Code to Reproduce
```
from art import *
print(text2art("Display Text", font='tarty2', chr_ignore=True))
```

#### Expected Behavior
Should print "Default text" in tarty2 font

#### Actual Behavior
```
print(text2art("Default text", font='tarty2', chr_ignore=True))
      ^^^^^^^^
NameError: name 'text2art' is not defined
```
#### Operating System
Windows 11
#### Python Version
3.11

#### ART Version (Use : `art.__version__`)
5.8

**Comment by sepandhaghighi** [comment]
@KrAsH-CoD3 Hi
It's a bit weird!
How do you install the `art` library?

**Comment by KrAsH-CoD3** [comment]
> @KrAsH-CoD3 Hi It's a bit weird! How do you install the `art` library?

`pip install art`. Also checking `pip list` art is listed.

**Comment by KrAsH-CoD3** [comment]
> > @KrAsH-CoD3 Hi It's a bit weird! How do you install the `art` library?
> 
> `pip install art`. Also checking `pip list` art is listed.

FIXED: Re-installed/Updated the package to 5.9
